### ISSUE TITLE
Update: Magister Wrapper

### ISSUE THREAD

**Issue by BramLeisink** [BramLeisink/rooster-documentation#7]
Ik ben momenteel bezig met het maken van een eerste versie van de frontend van Rooster. Deze versie heeft nog geen functionaliteiten, maar is een goed begin om andere mensen al snel om feedback te kunnen vragen.

## Feedback
Het is de bedoeling om dit prototype met zo veel mogelijk mensen te delen, om snel veel feedback te verzamelen. Het fijnste zou zijn om een volledig werkende versie te maken, maar dit is niet handig. Een volledig werkende site kan nog wel een jaar duren. Toch is er een alternatief om een werkend prototype te maken:

## Magister integratie 
Na wat zoeken, heb ik gevonden dat Magister een API heeft (een manier om de data op Magister te gebruiken). Deze gebruiken voor het prototype zou geweldig zijn: mensen loggen in met hun Magister account en zien daarna de Rooster site, maar met al hun informatie. Zo is er een zo realistisch mogelijk prototype, kunnen wij deze zelf gebruiken om te zien wat er verbeterd moet worden en kan dit meteen aangepast worden.

Er is helaas één probleem: je moet toestemming hebben om van de Magister API gebruik te maken. Ik ben bang dat als we aan komen met: "Hallo, wij proberen een gratis alternatief voor jullie te maken, mogen wij jullie API gebruiken?", dat we niet snel een "ja" krijgen.

Als we zo' n API wachtwoord willen aanvragen, moeten we allereerst de site aanleveren, zodat ze kunnen controleren of we veilig omgaan met gebruikers informatie. Dit hoeft geen probleem te zijn. Daarnaast is het noodzakelijk dat we deze repository, tijdelijk op prive zetten. We kunnen Rooster voordoen als een schoolproject (we zijn tenslotte Technasium) en zeggen dat we bezig zijn met een project waarbij we nadenken over hoe apps als Magister beter zouden kunnen. Wij zeggen dat we, omdat we Magister toch niet kunnen vervangen, we hebben besloten gewoon een nieuwe Magister interface te maken.

**Comment by QuintenC08** [comment]
maar magister API gebruiken is dan niet iets wat je wilt blijven doen dan toch? of wil je het long-term integreren?

**Comment by BramLeisink** [comment]
Nee, die is alleen om mensen enthausiast te maken voor Rooster. Later kan dit alles omgezet worden naar onze eigen Rooster-api.
